pkg/yqlib/lib.go

```diff
@@ -166,6 +166,24 @@ func NodeToString(node *CandidateNode) string {
 	return fmt.Sprintf(`D%v, P%v, (%v)::%v`, node.Document, node.Path, tag, buf.String())
 }
 
+// EvaluateNodes takes an expression and one or more yaml nodes, returning a list of matching candidate nodes
+func EvaluateNodes(expression string, nodes ...*yaml.Node) (*list.List, error) {
```

Comment:
Not sure what kind of testing pattern should go in `lib_testing.go` since these functions are not compatible with the `expressionScenario` pattern